pkg/agent/nodeportlocal/rules/iptable_rule.go

```diff
@@ -29,6 +30,13 @@ import (
 // NodePortLocalChain is the name of the chain in IPTABLES for Node Port Local
 const NodePortLocalChain = "ANTREA-NODE-PORT-LOCAL"
 
+// DestinationPodIPPort stores the PodIP and Pod port parsed from the iptable rules
+// for NPL
+type DestinationPodIPPort struct {
```

Comment:
There are some structs having similar structs but a little difference, for instance, `DestinationPodIPPort` has no `NodePort`, `PodNodePort` has no `PodIP`. It seems just because they are using different fields as keys in different places. Maybe we could just have one struct and let the key redundant in value to avoid declaring multiple structs to keep code cleaner.

Reply:
Makes sense. Will change these to have only struct.